cmd/kubernetes_version.go

```diff
@@ -42,6 +43,20 @@ func getKubeVersionFromRESTAPI() (string, error) {
 	return k8sVersion, nil
 }
 
+func getWebDataWithRetry(k8sVersionURL, token string, cacert *tls.Certificate) (data []byte, err error) {
+	tries := 0
+	for tries < 10 {
```

Comment:
```suggestion
      // We retry a few times in case the DNS service has not had time to come up
	for tries < 10 {
```

Suggest adding a comment so that we know why it's an arbitrary number of retries, and in case we ever need to increase that number